aurora_module/rds_aurora_cluster.tf

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) 2011 - 2019, Coveo Solutions Inc.
+ */
+
+locals {
+  instances_count = "${var.engine_mode == "serverless" ? 0 : var.instances_count}"
+}
+
+resource "aws_db_subnet_group" "db_subnet_group" {
+  subnet_ids = "${var.subnet_ids}"
+
+  name        = "${lookup(var.optional_parameters, "db_subnet_group_name", "${var.custom_identifier}")}"
+  description = "${lookup(var.optional_parameters, "subnet_group_description", "${var.custom_identifier}")}"
+  tags        = "${var.subnet_group_tags}"
+}
+
+resource "aws_kms_key" "rds_cluster_encryption_key" {
```

Comment:
I'm not sure everyone wants a KMS key per Cluster. We tend to create one KMS key per team, or one KMS key per "sensitivity level". Creating a KMS key could be an option? We could pass our own KMS key ID?

Reply:
Sure, I could add an option to pass your own kms key id.